src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDatabaseMetaData.java

```diff
@@ -25,6 +25,7 @@
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.ReentrantLock;
 import java.util.logging.Level;
+import com.microsoft.sqlserver.jdbc.JDBCType;
```

Comment:
Unused.
```suggestion
```

Reply:
^ this is not addressed. This import is not necessary

Reply:
Import has been removed